en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -1,24 +1,17 @@
-# Project Design Diagram
+# Project Design view
 
-With the increasing size and complexity, software applications have evolved to be a composition of multiple interrelated components. When the number and/or depth of such components grow, it becomes convenient to have a high-level picture of the system components and their interactions with one another. The Ballerina Project Design View is deployed with the intention of providing a convenient way to generate such a high-level picture of projects purely based on the source code. In addition to the component interactions, the Project Design View is also capable of visualizing the Ballerina record types in the project workspace and their associations.
+Software applications have evolved into a composition of multiple interrelated components to be able to handle increasing complexities. When the number and/or depth of such components grow, it is convenient to have a high-level picture of the system components and their interactions with one another. 
 
-## Set up the prerequisites
-
-Install the [latest Ballerina version](https://ballerina.io/downloads/) and [Ballerina Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=wso2.ballerina).
-
->**Info:** The Ballerina Project Design Diagram tool is shipped with the latest release of the Ballerina VS Code extension available in the VS Code marketplace. The diagrams can be generated upon executing the `Ballerina: Project Design` extension command. However, to get the best possible view of the project, it is necessary to set up the Ballerina project with the following concepts in mind.
+The Ballerina Project Design view is enabled in the VS Code extension to provide such a high-level picture of Ballerina projects purely based on the source code. In addition to the component interactions, the Project Design view is also capable of visualizing the Ballerina record types in the project workspace and their associations.
 
 ## Related concepts
 
-Basic concepts that are related to the project design diagram are listed below.
+When you create your Ballerina project, be mindful of the following concepts as they will affect the Project Design view.
 
-### Ballerina project
+### Ballerina project workspace
```

Comment:
It is not correct to combine the two topics `Ballerina project` and `Project workspace` as those two are two separate things.

Reply:
Got it. Will change it to the way it was.